README.md

```diff
@@ -97,93 +138,63 @@ END:VEVENT
 END:VCALENDAR
 """
 
-ntt_provider = init_provider("ntt")
-
 data_to_process = NotificationData.init_from_raw("ical", raw_data)
 
-maintenances = ntt_provider.get_maintenances(data_to_process)
+type(data_to_process)
+<class 'circuit_maintenance_parser.data.NotificationData'>
+```
+
+Finally, with we retrieve the maintenances (it is a `List` because a notification can contain multiple maintenances) from the data calling the `get_maintenances` method from the `Provider` instance:
+
+```python
+maintenances = generic_provider.get_maintenances(data_to_process)
 
 print(maintenances[0].to_json())
 {
-  "account": "137.035999173",
-  "circuits": [
-    {
-      "circuit_id": "acme-widgets-as-a-service",
-      "impact": "NO-IMPACT"
-    },
-    {
-      "circuit_id": "acme-widgets-as-a-service-2",
-      "impact": "OUTAGE"
-    }
-  ],
-  "end": 1444471200,
-  "maintenance_id": "WorkOrder-31415",
-  "organizer": "mailto:noone@example.com",
-  "provider": "example.com",
-  "sequence": 1,
-  "stamp": 1444435800,
-  "start": 1444464000,
-  "status": "TENTATIVE",
-  "summary": "Maint Note Example",
-  "uid": "42"
+"account": "137.035999173",
+"circuits": [
+{
+"circuit_id": "acme-widgets-as-a-service",
+"impact": "NO-IMPACT"
+},
+{
+"circuit_id": "acme-widgets-as-a-service-2",
+"impact": "OUTAGE"
+}
+],
+"end": 1444471200,
+"maintenance_id": "WorkOrder-31415",
+"organizer": "mailto:noone@example.com",
+"provider": "example.com",
+"sequence": 1,
+"stamp": 1444435800,
+"start": 1444464000,
+"status": "TENTATIVE",
+"summary": "Maint Note Example",
+"uid": "42"
 }
 ```
 
-## CLI
+Notice that, either with the `GenericProvider` or `NTT` provider, we get the same result from the same data, because they are using exactly the same `Processor` and `Parser`. The only difference is that `NTT` notifications come without `organizer` and `provider` in the notification, and this info is fulfilled with some default values for the `Provider`, but in this case the original notification contains all the necessary information, so the defaults are not used.
```

Comment:
Maybe not a great example here? If NTT is the same as GenericProvider, why would I as a user ever want to use the NTT provider specifically? What purpose does it serve?

Reply:
I used this example to show that some Providers (such as NTT, PacketFabric,) are using the default iCalendar parser and only add some custom default data (_default_organizer) to extend notifications data that maybe don't contain all the mandatory attributes.